app/views/admin/merchants/show.html.erb

```diff
@@ -3,7 +3,7 @@
 <p class="address"><%= @merchant.address %></p>
 <p class="address"><%= @merchant.city %>, <%= @merchant.state %> <%= @merchant.zip %></p>
 
-<h3><%= link_to "View All My Items", "/merchant/items" %><h3>
+<h3><%= link_to "View All My Items", "/merchants/#{@merchant.id}/items" %><h3>
```

Comment:
now when an admin wants to see a merchant's items from the admin's merchant dashboard it doesn't break things!

Reply:
And I wouldve gotten away with it too if it werent for those DEVS

Reply:
![](https://media0.giphy.com/media/B2RwHaB7pehJcQsVjK/giphy.gif?cid=5a38a5a261eef5faa10fdc4426d2b6294235fcbc391738a2&rid=giphy.gif)